theforeman.org/yaml/jobs/internal_folder.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- job:
+  name: internal-folder
```

Comment:
jenkins_jobs.errors.JenkinsJobsException: Syntax error, for item named 'internal-folder'. Missing indent?